WordPressAuthenticator/Unified Auth/View Related/Password/PasswordViewController.swift

```diff
@@ -61,6 +61,8 @@ class PasswordViewController: LoginViewController {
         
         if isMovingToParent {
             tracker.track(step: .userPasswordScreenShown)
+        } else {
+            tracker.set(step: .userPasswordScreenShown)
         }
```

Comment:
Unless I'm misunderstanding, you can remove the if/else statement and just use ` tracker.set(step: .userPasswordScreenShown)`